### DISCUSSION TITLE
Feed looks cluttered with all those recommendations.

### DISCUSSION THREAD

**Discussion by edenroseFR** [community/community#13464] [Other Feature Feedback, Questions, & Ideas] [answered]
The only good thing in this update is the small emoji on the bottom right side of the user, which makes it easier to grasp the kind of activity the user has done.

I'd like the recommended repositories to just stay on the `Explore repositories` section. Also, remove the recommended people based on who I follow because it's unnecessary.

**Comment by andre-st** [comment]
True. It all seems very confused, not properly aligned, no clear visual hierarchies to direct a viewer’s eye / randomly sized icons, tons of unnecessary repetition. Probably horror for people with OCD.
![repetition](https://user-images.githubusercontent.com/26840833/161434752-f5541dd4-4b5e-4dda-a448-34a60f4a6ec0.png)
![ballpit](https://user-images.githubusercontent.com/26840833/161452403-f925e94c-e579-438b-8aab-62542d373f39.png)

**Alternative:**
Assuming that all of this feed information is desired, it might be better, at least for presentation, to group things into (non-repeating) sections:

```
RECOMMENDED USERS: 
Avatar, Avatar, Avatar, ...

LATEST RELEASES: 
Project     [more info v], 
Project     [more info v],
...
```

**Overall:**
Long strings of disjointed, isolated chunks of information with lots of repetition etc. is actually something you show computers, not people. 
For social media, this display problem is harder to solve than for _defined events_ from Github. Whether the Feed/Timeline display type was the best choice here is questionable.

**Comment by jfullstackdev** [comment, marked as answer]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
